v1/index.ts

```diff
@@ -0,0 +1,12 @@
+import apiBuilder from './api'
+
+const PORT = Number(process.env.PORT ?? '8080')
+
+apiBuilder({ useLogging: true, useSwagger: true })
```

Comment:
I think we can use top level await if we target nodejs 18 with native ESM. Mind reworking around that?